### DISCUSSION TITLE
How to build a PWA in vue-cli mode?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#9656] [CLI - PWA mode]
I built in the past PWAs though `quasar build --mode pwa`. The [documentation](https://quasar.dev/quasar-cli/developing-pwa/build-commands), however, does not tell **how to build it when the project was done based on `vue-cli`**.

I currently build my app with `npm run build` .

**Comment by metalsadman** [comment]
it's only available for quasar cli.

**Reply by ontwikkelfabriek** [reply]
@metalsadman  @wsw70 

> it's only available for quasar cli.

Here in the 'flavour' comparison table is **does** say it is possible with vue-cli:
`Build SPA, PWA: 3X yes`
https://quasar.dev/start/pick-quasar-flavour
